docs/examples.md

```diff
@@ -0,0 +1,17 @@
+# Examples
+
+## Custom logging
+
+The below `logging_example.yaml` is a yaml representation of the default logging configuration.
+You can pass it loading a dict, using the `--log-config` flag.
```

Comment:
Just checking... Does this definately work?
I haven't checked out if Python's log file format also supports using the yaml style?

Reply:
this is definitely an ambiguous formulation, python logging supports dict / file (ini style). A lot of time people use yaml and load it to dict with pyyaml for instance, that's why I said "representation"

Reply:
Right, but it wouldn't directly work with our `--log-config` flag right?

(Tho we *could* potentially change things here, so that we *expect* a YAML style logging config file.)

Reply:
correct, one would have to do something like
```
    with open('logging_example.yaml', 'r') as stream:
        config = yaml.load(stream, yaml.FullLoader)
    uvicorn.run("main:app", reload=True, log_config=config, log_level="trace")
```